tests/e2e/test_api.py

```diff
@@ -590,6 +590,7 @@ def test_metrics() -> None:
         "llm_validation_errors_total",
         "llm_token_sent_total",
         "llm_token_received_total",
+        "model_enabled",
```

Comment:
do we not have a test for the selected provider/model metrics?

Reply:
I need to figure out if it will work in all circumstances. I'll look at it, definitely (was on my TODO already)

Reply:
ok works everywhere, added @bparees

Reply:
(probably later we can add e2e tests for specific CI environments, but I'd prefer separate PR)